tests/__init__.py

```diff
@@ -0,0 +1,31 @@
+import os.path
+import unittest
+
+import s5a
+
+TEST_FILE = os.path.join(
+    os.path.dirname(os.path.abspath(__file__)),
+    'test.nc')
+
+
+class TestS5a(unittest.TestCase):
+    def setUp(self):
+        self.data = s5a.load_ncfile(TEST_FILE)
+        self.assertEqual(self.data.size, 1130)
+
+    def test_quality(self):
+        # tuples of quality and number of points
+        tests = (
+            (0, 1130),
+            (0.5, 900),
+            (1, 0),
+        )
+        for test in tests:
```

Comment:
Maybe something like
```suggestion
        for quality, expected_result in tests:
```